py/torch_tensorrt/dynamo/conversion/_TRTInterpreter.py

```diff
@@ -477,12 +477,18 @@ def _save_weight_mapping(self) -> None:
                     # Retrieve each weight name(s) in state_dict
                     if layer_type == "CONSTANT":
                         if "embedding" in suffix:
-                            sd_weight_name = f"{sd_weight_name}.{torch_attr[0]}"
```

Comment:
@zewenli98 keep track of this, seems like there could be a lot of possible names we need to have might want to look at a generic solution later